src/components/messenger/list/group-details-panel/index.tsx

```diff
@@ -28,24 +27,25 @@ export class GroupDetailsPanel extends React.Component<Properties, State> {
   state = { name: '', image: null };
 
   createGroup = () => {
-    this.props.onCreate({ name: this.state.name, users: this.props.users, image: this.state.image });
+    if (this.isNameValid) {
+      this.props.onCreate({ name: this.state.name, users: this.props.users, image: this.state.image });
+    }
```

Comment:
Ideally we shouldn't need this check. `createGroup` should never be called if the name is empty (since the button is disabled)